src/jni_core/src/jvm_runtime.rs

```diff
@@ -100,7 +100,14 @@ impl JavaVmWrapper {
             .option("-Dis_embedded_connector=true")
             .option(format!("-Djava.class.path={}", class_vec.join(":")))
             .option("-Xms16m")
-            .option(format!("-Xmx{}", jvm_heap_size));
+            .option(format!("-Xmx{}", jvm_heap_size))
+            // Quoted from the debezium document:
+            // > Your application should always properly stop the engine to ensure graceful and complete
+            // > shutdown and that each source record is sent to the application exactly one time.
+            // In RisingWave we assume the upstream changelog may contain duplicate events and
+            // handle conflicts in the mview operator, thus we don't need to obey the above
+            // instructions. So we decrease the wait time here to reclaim jvm thread faster.
+            .option("-Ddebezium.embedded.shutdown.pause.before.interrupt.ms=1");
```

Comment:
btw can you share more about the param?

Reply:
It is a  timeout config to forcefully stop the engine thread.
https://github.com/debezium/debezium/blob/e67620d30ee12ce2c257b2fad2eca4259eaef0e6/debezium-embedded/src/main/java/io/debezium/embedded/EmbeddedEngine.java#L240-L247